sdk/search/Azure.Search.Documents/src/SearchClient.cs

```diff
@@ -522,11 +395,11 @@ public class SearchClient
         /// Thrown when a failure is returned by the Search Service.
         /// </exception>
         /// <remarks>
-        /// The generic overloads of the GetDocument and GetDocumentAsync
-        /// methods support mapping of Azure Search field types to .NET types
-        /// via the type parameter T.  Note that all search field types except
-        /// collections are nullable, so we recommend using nullable types for
-        /// the properties of T.  The type mapping is as follows:
+        /// The GetDocument and GetDocumentAsync methods support mapping of
```

Comment:
I didn't for this one because you're already on that doc page, but I guess you could theoretically want to jump over to the other.  I'll change some of these now where I notice them, but we should come up with some lightweight rules and fix them in a big sweep as part of #10612.